generators/generator-bot-adaptive/tests/generator.test.js

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+function importTests(name, path) {
+  describe(name, function () {
+    require(path);
+  });
+}
+
+describe('generator-bot-adaptive', function () {
```

Comment:
I think `mocha` should recursively discover everything in the `test` directory. This pattern of registering tests in a single file is a little unusual. Generally, each test would `describe` its own hierarchy of tests.

All four tests are fairly common now, but we might want to add other language-specific tests. In my (humble) opinion, this pattern doesn't have enough shared utility to warrant the non-standard structure that may impact our ability to iterate on the tests in the future.

Reply:
I've moved all tests up to a `test` dir and removed this file.